client/app/models/project.js

```diff
@@ -143,24 +142,18 @@ export default class ProjectModel extends Model {
   }
 
   @computed('milestones')
-  get filteredMilestones() {
```

Comment:
Hmm, since we were intentionally filtering it out, I bet there was a previous requirement to filter it out. But if ZAP team is sure they want it again, then sounds good removing the filter...